crc32.c

```diff
@@ -344,14 +320,14 @@ int has_pclmul(void) {
  * "Fast CRC Computation for Generic Polynomials Using PCLMULQDQ Instruction"
  *  V. Gopal, E. Ozturk, et al., 2009, http://intel.ly/2ySEwL0
  */
-                                
+                         
 #ifdef _MSC_VER
 #define zalign(x) __declspec(align(x))
 #else
 #define zalign(x) __attribute__((aligned((x))))
 #endif
 
-uLong crc32_simd(unsigned char const *buf, size_t len, uInt crc) {
+uint crc32_simd(unsigned char const *buf, size_t len, uInt crc) {
```

Comment:
I would rather use `uint32_t` everywhere the size is known to be 32 bits.

Reply:
fixed (https://github.com/rordenlab/zlib/commit/59c0c27261a0c20c9e751b71f0ecce6cb352385b)